app/app/explore/components/StatusModal.tsx

```diff
@@ -0,0 +1,293 @@
+import { addDollarSignAndSuffix, getLogoPath } from "@/app/utils";
+import { PoolDetail } from "@/midgard";
+import {
+  PositionStatus,
+  PositionType,
+} from "@/utils/lp-monitor/parsePositions";
+import { Asset, AssetAmount, assetFromString } from "@xchainjs/xchain-util";
+import React, { useEffect, useRef, useState } from "react";
+import Image from "next/image";
+import { useAppState } from "@/utils/contexts/context";
+import { ChainKey } from "@/utils/wallet/constants";
+import {
+  LpSubstepsAddLiquidity,
+  useLiquidityPosition,
+} from "@/hooks/useLiquidityPosition";
+import { useLiquidityPositions } from "@/utils/contexts/PositionsContext";
+import LpSubstepDetail from "./LpSubstepDetail";
+
+export interface StatusStepData {
+  pool: PoolDetail;
+  assetAmount: AssetAmount;
+  assetUsdAmount: number;
+  runeAmount: AssetAmount;
+  runeUsdAmount: number;
+  positionType: PositionType;
+  neccessarySteps: LpSubstepsAddLiquidity[];
```

Comment:
please rename to `necessarySteps` or maybe `requiredSteps`, or even just `steps`.